shell/browser/native_window_mac.mm

```diff
@@ -609,16 +609,20 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
 }
 
 bool NativeWindowMac::IsMaximized() {
-  if (([window_ styleMask] & NSWindowStyleMaskResizable) != 0) {
+  if (([window_ styleMask] & NSWindowStyleMaskResizable) != 0)
```

Comment:
It does not seem to take this situation into account.

Reply:
@BlackHole1 this issue doesn't occur if the window is resizable.